elastic/apm-server
# beater: run gmuxed gRPC server by default

## Motivation/summary

Use gmux to run a gRPC server on the same port as the Elastic APM HTTP server, and register the Jaeger services by default.

When auth is enabled, it is necessary to configure Jaeger to send an agent tag called "elastic-apm-auth".
This cannot be changed.

## Checklist

~- [ ] Documentation has been updated~ Let's do this in a followup, after we have a discussion about the UX around auth tags.

## How to test these changes

- Run APM Server with default config
- Run `jaeger-agent` with `--reporter.grpc.host-port=localhost:8200`
- Exercise an app instrumented with the Jaeger client
- Check that Jaeger spans are indexed

## Related issues

Closes #3984
